### ISSUE TITLE
Unable to run code blocks that are already indented

### ISSUE THREAD

**Issue by tobiasbj** [nteract/hydrogen#1285]
**Description:**

Unable to run code blocks that are already indented.
Example is shown below.

**Steps to Reproduce:**

1. Copy paste this code.
```
for i in range(10):  # This is the line you ignore
    test = "test"
    if test is None:
        print("yes")
    else:
        print("no")
```

2. Mark and run all lines except the initial one.

**Versions:**

Windows 10
Atom 1.25.1
Hydrogen 2.3.0

**Logs:**
Error output
```
File "<ipython-input-1-ec679601011f>", line 2
    if test is None:
    ^
IndentationError: unexpected indent
```

**Comment by tobiasbj** [comment]
Ah, cool. No I haven't tried hydrogen-python yet. I'll try it now 👍

**Comment by BenRussert** [comment]
Looks like this was fixed in #1363, try this again after you update hydrogen. Feel free to reply if you still face the issue after updating.
